codacy/values-microk8s.yaml

```diff
@@ -0,0 +1,266 @@
+    create: false
+    postgresqlUsername: <--- db username --->
+    postgresqlDatabase: <--- db name --->
+    postgresqlPassword: <--- db password --->
+    host: <--- db url --->
+    service:
+      port: <--- db port --->
+  metricsdb:
+    create: false
+    postgresqlUsername: <--- db username --->
+    postgresqlDatabase: <--- db name --->
+    postgresqlPassword: <--- db password --->
+    host: <--- db url --->
+    service:
+      port: <--- db port --->
+  filestoredb:
+    create: false
+    postgresqlUsername: <--- db username --->
+    postgresqlDatabase: <--- db name --->
+    postgresqlPassword: <--- db password --->
+    host: <--- db url --->
+    service:
+      port: <--- db port --->
+  jobsdb:
+    create: false
+    postgresqlUsername: <--- db username --->
+    postgresqlDatabase: <--- db name --->
+    postgresqlPassword: <--- db password --->
+    host: <--- db url --->
+    service:
+      port: <--- db port --->
+codacy-api:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 1000m
+      memory: 2000Mi
+    requests:
+      cpu: 500m
+      memory: 500Mi
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: "nginx"
+    hosts:
+    - host: '<--- codacy url --->'
+      paths:
+        - /
+portal:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 500Mi
+    requests:
+      cpu: 200m
+      memory: 200Mi
+activities:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 750Mi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  activitiesdb:
+    create: false
+    postgresqlUsername: <--- db username --->
+    postgresqlDatabase: <--- db name --->
+    postgresqlPassword: <--- db password --->
+    host: <--- db url --->
+    service:
+      port: <--- db port --->
+remote-provider-service:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 500Mi
+    requests:
+      cpu: 200m
+      memory: 200Mi
+hotspots-api:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 500Mi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  hotspotsdb:
+    create: false
+    postgresqlUsername: <--- db username --->
+    postgresqlDatabase: <--- db name --->
+    postgresqlPassword: <--- db password --->
+    host: <--- db url --->
+    service:
+      port: <--- db port --->
+hotspots-worker:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 500Mi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+listener:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 1000Mi
+    requests:
+      cpu: 200m
+      memory: 1000Mi
+  listenerdb:
+    create: false
+    postgresqlUsername: <--- db username --->
+    postgresqlDatabase: <--- db name --->
+    postgresqlPassword: <--- db password --->
+    host: <--- db url --->
+    service:
+      port: <--- db port --->
+  persistence:
+    claim:
+      size: 50Gi
+  nfsserverprovisioner:
```

Comment:
we are missing to set requests and limits here, this is quite important:
```
  nfsserverprovisioner:
    enabled: true
    persistence:
      enabled: true
      size: 100Gi
    resources:
      limits:
        cpu: 250m
        memory: 200Mi
      requests:
        cpu: 100m
        memory: 200Mi
```
is my current local configuration, I don't know if this has impact on the analysis or not.